lib/auth/proxywatcher.go

```diff
@@ -29,10 +30,6 @@ import (
 	log "github.com/sirupsen/logrus"
 )
 
-// NewProxyWatcherFunc creates a new instance of proxy watcher,
-// used in tests
-type NewProxyWatcherFunc func() (*ProxyWatcher, error)
```

Comment:
Unnecessary as the proxy watcher test is used in that specific [test](https://github.com/gravitational/teleport/blob/dmitri/split-client-server/lib/auth/local/local_test.go#L913) and had no place in a suite